.drone.yml

```diff
@@ -0,0 +1,157 @@
+workspace:
+  base: /var/www/owncloud
+  path: apps/notifications
+
+branches: [master]
+
+pipeline:
+  install-server:
+    image: owncloudci/core
+    pull: true
+    git_reference: ${OC_VERSION}
+
+  install-app:
+    image: owncloudci/php:${PHP_VERSION}
+    pull: true
+    commands:
+      - cd /var/www/owncloud/
+      - php occ a:l
+      - php occ a:e testing
+      - php occ a:l
+      - php occ config:system:set trusted_domains 1 --value=owncloud
+      - php occ log:manage --level 0
+
+  owncloud-log:
+    image: owncloud/ubuntu:16.04
+    detach: true
+    pull: true
+    commands:
+      - tail -f /var/www/owncloud/data/owncloud.log
+
+  code-compliance-check:
+    image: owncloudci/php:${PHP_VERSION}
+    pull: true
+    commands:
+      - cd /var/www/owncloud/apps/notifications
+      - ls -l
+      - cat Makefile
+      - make test-php-codecheck
+      - make test-php-lint
+
+  fix-permissions:
+    image: owncloudci/php:${PHP_VERSION}
+    pull: true
+    commands:
+      - cd /var/www/owncloud/
+      - chown www-data * -R
+
+  webui-acceptance-tests:
+    image: owncloudci/php:${PHP_VERSION}
+    pull: true
+    environment:
+      - BROWSER=chrome #chrome or firefox
+      - SELENIUM_HOST=selenium
+      - SRV_HOST_NAME=owncloud
+      - SRV_HOST_PORT=80
+      - SKELETON_DIR=/var/www/owncloud/tests/acceptance/webUISkeleton
+      - SELENIUM_PORT=4444
+      - PLATFORM=Linux
+      - APPS_TO_ENABLE=notifications
+    commands:
+      - cd /var/www/owncloud
+      - chmod 777 /var/www/owncloud/tests/acceptance/filesForUpload -R
+      - bash tests/travis/start_ui_tests.sh --remote --config /var/www/owncloud/apps/notifications/tests/acceptance/config/behat.yml
+    when:
+      matrix:
+        TEST_SUITE: web-acceptance
+
+  api-acceptance-tests:
+    image: owncloudci/php:${PHP_VERSION}
+    pull: true
+    commands:
+      - cd /var/www/owncloud/apps/notifications/tests/acceptance
+      - bash ./run.sh apiNotifications
+    when:
+      matrix:
+        TEST_SUITE: api-acceptance
+
+  phpunit-tests:
+    image: owncloudci/php:${PHP_VERSION}
+    pull: true
+    environment:
+      - PHP_VERSION=${PHP_VERSION}
+    commands:
+      - cd /var/www/owncloud
+      - phpdbg -d memory_limit=4096M -rr ./lib/composer/bin/phpunit --configuration apps/notifications/tests/Unit/phpunit.xml
+      - cd apps/notifications/
+      - wget -q https://scrutinizer-ci.com/ocular.phar
+      - php ocular.phar code-coverage:upload --format=php-clover tests/Unit/clover.xml
+    when:
+      matrix:
+        TEST_SUITE: phpunit
+
+  codecov:
```

Comment:
please use the official plugin:
https://github.com/owncloud/core/blob/master/.drone.yml#L96

token will be set as secret and then can be used

Reply:
huh - but the step is `codecov` and uses `robertstettner/drone-codecov` plugin 

- any tipps on multiline selection for PR reviews? maybe that would have made it clearer